collector/src/runtime.rs

```diff
@@ -99,7 +145,8 @@ fn compile_runtime_benchmarks(toolchain: &LocalToolchain, dir: &Path) -> anyhow:
 }
 
 /// Parse Cargo JSON output and find all compiled binaries.
-fn gather_binaries(cargo_stdout: &[u8]) -> anyhow::Result<Vec<BenchmarkBinary>> {
+/// Then execute each benchmark with the `list-benchmarks` command to find out its benchmark names.
```

Comment:
Does "benchmark" have more than one meaning in this sentence? I don't understand how a benchmark can have multiple benchmark names.